docs/openapi.yml

```diff
@@ -309,6 +356,17 @@ components:
         type: string
         format: uuid
 
+    datasourceUrn:
+      name: datasourceUrn
+      in: path
+      description: The URN of the datasource.
+      required: true
+      schema:
+        type: string
+        format: URN
+        maxLength: 1024
+        example: urn:datasource:jdbcpostgresqllocalhost5431novelists834:Datasource2168
```

Comment:
Minor: This most likely isn't the URN returned by the API if you provide:

```
{
  "name": "Datasource2168",
  "connectionUrl": "jdbc:postgresql://localhost:5431/novelists_834"
}
```

Mind verifying the response?

Reply:
Ran the inputs/outputs through a locally-running instance of Marquez and updated the results.

Perhaps a nit, but I also think that hostname `localhost` may not make a lot of sense here, just b/c we our storage of datastore to have a universally-identifiable host (so it can be repro'd, audited, etc.). If folks think it's worth changing, I can change it.